staff/clara-rufi/mailchimp-template/style.css

```diff
@@ -0,0 +1,274 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto");
+ul, li {
+  list-style: none;
+}
+
+h3 {
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  background: #46BDD4;
+  flex-flow: column;
+  align-items: flex-start;
+  display: flex;
+  top: 0px;
+}
+
+.headerchimptop {
+  display: flex;
+  flex-direction: row;
+  flex-flow: row;
+  margin-top: 1.2rem;
+  margin-left: 70rem;
+  font-size: 1.2rem;
+  color: white;
+}
+
+.mailchimplogo {
+  display: flex;
+  align-items: flex-start;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.headerchimp {
+  background: #46BDD4;
+}
+
+.logomonkey {
+  margin-left: 6rem;
+  margin-top: 1rem;
+  margin-right: 2rem;
+  width: 90%;
+}
+
+.login {
+  display: flex;
+  justify-content: flex-end;
+  margin: 1.5rem 4.5rem;
+  color: white;
+  font-size: 1.2rem;
+}
+
+.navmenu {
+  display: inline-flex;
+  justify-content: center;
+  margin-top: 4rem;
+}
+.navmenu li {
+  margin: 0 0.9rem;
+}
+.navmenu a {
+  color: #494646;
+  text-decoration: none;
+  font-weight: 600;
+}
+.navmenu_logo {
+  margin-left: 8rem;
+  margin-top: -1.5rem;
+}
+
+.logosignup_red {
+  background: #DF7045;
+  width: 14rem;
+  height: 4rem;
+  border-radius: 0.6rem;
+  color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 1rem;
+}
+
+.logosignup_black {
+  background: #465255;
+  width: 7.5rem;
+  height: 4rem;
+  border-radius: 0.6rem;
+  color: white;
+  display: flex;
```

Comment:
You are repeating `display: flex`, `align-items: center`, `margin-top: -2rem`, etc...
You can either define all of this in one class and repeat the class itself in the html file or use `@extend %...` in each place where you want this parameters.